src/renderer/api/__tests__/catalog-entity-registry.test.ts

```diff
@@ -43,27 +43,6 @@ describe("CatalogEntityRegistry", () => {
       expect(catalog.items.length).toEqual(2);
     });
 
-    it("ignores unknown items", () => {
```

Comment:
Why remove? Makes no sense anymore?

Reply:
Because I removed the merge logic, so the `renderer` just takes the whole blast from `main` and uses that.